### ISSUE TITLE
Support local directories with a package.json

### ISSUE THREAD

**Issue by sschuberth** [npm/npm-remote-ls#31]
When working on the source code of a module, it would be nice if `npm-remote-ls` was able to list the transitive dependencies of that local module by pointing it at the directory containing the `package.json` file.

**Comment by toddpi314** [comment]
This could be done by just seeding the package.json object into the task description. 

In `_loadPackageJson`, you could just check to see if there is a default object and skip the registry fetch to directly call `_walkDependencies`.

I am very often in a situation where I want to `npm ls` without suffering the entire `npm install` on a directory.

**Comment by robross0606** [comment]
Agreed, this would be immensely useful so we can start from a local `package.json` and do an `npm ls` starting from there without traversing into a repo first.  This would be especially helpful when you want to start from packages which are deployed only to something like a container and never stored in a repo, but use packages which _are_ stored in a repo.
